src/data/reducers/loginForm.js

```diff
@@ -0,0 +1,35 @@
+import {
+  LOGIN_FAILURE,
+  LOGIN_REQUEST,
+  LOGIN_SUCCESS,
+} from '../constants/loginForm';
+
+const loginForm = (state = {
+  isAuthenticated: localStorage.getItem('access_token') !== false,
+  isLoginSuccess: false,
```

Comment:
I think having just `isAuthenticated` is fine (no need for `isLoginSuccess` as you're setting both keys to the same value anyways).